server/data/data-file/src/test/java/org/apache/james/rrt/file/XMLRecipientRewriteTableTest.java

```diff
@@ -119,6 +119,8 @@ protected void removeMapping(String user, Domain domain, String mapping, Type ty
             mappings = mappings.remove(MappingImpl.address(mapping));
         } else if (type == Type.Domain) {
             mappings = mappings.remove(MappingImpl.domain(Domain.of(mapping)));
+        } else if (type == Type.Forward) {
```

Comment:
Switch case?

Reply:
I refactored that class and dropped huge chunk of code + simplified logic in it...